version

```diff
@@ -0,0 +1 @@
+2.11.0
```

Comment:
does this version mean anything, just curious?

Reply:
yeah, official nf/core version